static/home/js/selection-autocomplete.js

```diff
@@ -20,62 +20,86 @@ $.widget( "custom.catcomplete", $.ui.autocomplete, {
     }
   });
 var redirect_on_select_new_scope = redirect_on_select;
-$(function() {
-    $("#selection-autocomplete").catcomplete({
-        source: "/protein/autocomplete?selection_only_receptors="+selection_only_receptors+"&type_of_selection=" + type_of_selection,
-        minLength: 2,
-        autoFocus: true,
-        delay: 500,
-        create: function(event, ui) { this.focus();return false; },
-        focus: function(event, ui) { return false; },
-        select: function(event, ui) {
-            $( "#selection-autocomplete" ).val("");
+var selection_autocomplete_id_suffix = '';
```

Comment:
is this the same autocomplete function that is used in the navbar? If yes, please test that that one is still working as intended

Reply:
It is not, but thank you for reminding me to check other selection-autocomplete, because I broke them due to an undefined variable. They are now fixed by [0c3bf95](https://github.com/protwis/protwis/pull/550/commits/0c3bf95740150fd6c39619aa8620e8768136a842).